cache.rst

```diff
@@ -441,31 +423,17 @@ case the value needs to be recalculated.
             'cache' => [
                 'pools' => [
                     'my_cache_pool' => [
-                        'adapter' => 'cache.adapter.psr6',
-                        'provider' => 'app.my_cache_chain_adapter',
-                    ],
-                    'cache.my_redis' => [
-                        'adapter' => 'cache.adapter.redis',
-                        'provider' => 'redis://user:password@example.com',
-                    ],
-                    'cache.apcu' => [
-                        'adapter' => 'cache.adapter.apcu',
-                    ],
-                    'cache.array' => [
-                        'adapter' => 'cache.adapter.array',
+                        'default_lifetime' => 31536000, // One year
+                        'adapters' => [
+                            'cache.adapter.array',
+                            'cache.adapter.apcu',
+                            'redis://user:password@example.com' => 'cache.adapter.redis',
```

Comment:
FYI, the provider can be given as key,
alternatively it can be provided exactly as yaml does it:
`['name' => 'cache.adapter.redis', 'provider' => 'redis://...']`